packages/react-next/src/components/Checkbox/Checkbox.base.tsx

```diff
@@ -15,30 +13,20 @@ const defaultSlots: Omit<ICheckboxSlots, 'root'> = {
 
 export const CheckboxBase = compose<'div', ICheckboxProps, {}, ICheckboxProps, {}>(
   (props, forwardedRef, composeOptions) => {
-    const { slotProps, slots, state } = mergeProps<ICheckboxProps, ICheckboxState, ICheckboxSlots, ICheckboxSlotProps>(
+    const { slotProps, slots } = mergeProps<ICheckboxProps, ICheckboxState, ICheckboxSlots, ICheckboxSlotProps>(
       composeOptions.state,
       composeOptions,
     );
 
-    const { disabled, keytipProps } = state;
-
     const onRenderLabel = (): JSX.Element => {
       return <slots.label {...slotProps.label} />;
     };
 
-    // tslint:disable-next-line:no-any
-    const renderContent = (keytipAttributes: any = {}) => (
+    return (
       <slots.root {...slotProps.root}>
-        <slots.input
-          {...slotProps.input}
-          data-ktp-execute-target={keytipAttributes['data-ktp-execute-target']}
-          aria-describedby={mergeAriaAttributeValues(
-            slotProps.input['aria-describedby'],
-            keytipAttributes['aria-describedby'],
-          )}
-        />
+        <slots.input {...slotProps.input} />
         <slots.container {...slotProps.container}>
-          <slots.checkbox {...slotProps.checkbox} data-ktp-target={keytipAttributes['data-ktp-target']}>
```

Comment:
does this mean that this data- attribute no longer gets put on the element?

Reply:
it still is! and exactly the same attributes as what we have currently. code is here: https://github.com/microsoft/fluentui/pull/13742/files#diff-b2490de9a8f87eb3eb767dcfd41a69ed